.basic-auth-env

```diff
@@ -0,0 +1,27 @@
+# Disable OIDC
+quarkus.oidc.tenant-enabled=false
+
+# Enable Basic Auth
+quarkus.http.auth.basic=true
+
+# Enable Role Based Auth
+apicurio.auth.role-based-authorization=true
+
+# Enable Testing Embedded Users
+# TODO: check but should be redundant
+# quarkus.security.users.embedded.enabled=true
+quarkus.security.users.embedded.plain-text=true
+
+# Users configuration
+# Alice: "Authorization: Basic YWxpY2U6YWxpY2U="
+quarkus.security.users.embedded.users.alice=alice
+# Bob: "Authorization: Basic Ym9iOmJvYg=="
+# example:
+# curl -H "Authorization: Basic $(echo -n "alice:alice" | base64)" http://localhost:8080/apis/registry/v2/admin/rules -v
+quarkus.security.users.embedded.users.bob=bob
+quarkus.security.users.embedded.roles.alice=sr-admin
+quarkus.security.users.embedded.roles.bob=user
```

Comment:
Bob should probably have role `sr-developer`.

Reply:
improved over this example file.